clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/deploy/description/BasicAmazonDeployDescription.groovy

```diff
@@ -55,7 +55,7 @@ class BasicAmazonDeployDescription extends AbstractAmazonCredentialsDescription
   /**
    * When set to true, the created server group will use a launch template instead of a launch configuration.
    */
-  Boolean setLaunchTemplate = false
+  Boolean setLaunchTemplate = true
```

Comment:
defaulting to true and serving as "force launch config" if set to false in the request.

This allows a user to opt out when this feature is rolled out to all accounts and regions.